lib/models/tts.dart

```diff
@@ -42,4 +57,16 @@ class TtsModel {
       _tts.stop();
     }
   }
+
+  bool get isBotMuted {
+    return _isBotMuted;
+  }
+
+  set isBotMuted(bool value) {
```

Comment:
The value doesn't update because `TtsModel` doesn't implement `ChangeNotifier`.

To keep it a bit simpler (because I totally did that as a shortcut and not really thinking ahead...), instead of extracting `TtsModel` use the `ChatHistoryModel` and put a getter/setter on it that sets the property on `TtsModel`.

Basically a property like `ttsEnabled`: https://github.com/muxfd/rtchat/blob/main/lib/models/chat_history.dart#L159